ProcessLib/BoundaryCondition/DirichletBoundaryCondition.cpp

```diff
@@ -45,14 +51,17 @@ void DirichletBoundaryCondition::getEssentialBCValues(
         if (g_idx >= 0) {
             bc_values.ids.emplace_back(g_idx);
             bc_values.values.emplace_back(_parameter(t, pos).front());
+            DBUG("t = %g; Dirichlet at gidx %d value, %g", t, g_idx,
+                 _parameter(t, pos).front());
```

Comment:
That might produce excessive output in debug mode. Should it really be kept?

Reply:
Oh no.